src/styles/Login.scss

```diff
@@ -62,82 +62,83 @@
 
 @media only screen and (max-width: 1024px) {
   .login-page-content-container {
-    text-align: center;
     align-items: center;
+    text-align: center;
     width: 100%;
   }
 }
 
 .login-page-images {
-  width: 100%;
-  height: 100%;
+  align-items: center;
   display: flex;
+  height: 100%;
   justify-content: center;
-  align-items: center;
+  width: 100%;
 }
 
 .login-page-art {
+  max-height: 500px;
   position: absolute;
-  top: 120px;
   right: 30px;
+  top: 120px;
   width: 1000px;
-  max-height: 500px;
   z-index: 1;
 
   @media only screen and (max-width: 1024px) {
+    max-height: 200px;
+    max-width: 300px;
     position: relative;
-    top: -30px;
     right: 0;
-    max-width: 300px;
-    max-height: 200px;
+    top: -30px;
   }
 }
 
 .background-svg {
-  max-width: 1084px;
   height: 100%;
+  max-width: 1084px;
   position: absolute;
-  top: 0;
   right: 0;
+  top: 0;
 
   @media only screen and (max-width: 1024px) {
     display: none;
   }
 }
 
 .login-form-input {
-  font-size: 1.5em;
-  line-height: normal;
-  padding: 0.2rem 0.5rem;
-  border: 0;
-  border-radius: 8px;
+  appearance: none;
+  appearance: none;
+  appearance: none;
```

Comment:
```suggestion
  appearance: none;
```

Reply:
huh im kinda sus of all these changes, ill take a look at whats happening